packages/@angular/cli/models/webpack-configs/styles.ts

```diff
@@ -199,8 +199,12 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
       const ret: any = {
         include: globalStylePaths,
         test,
-        use: buildOptions.extractCss ? ExtractTextPlugin.extract(extractTextPlugin)
-                                     : ['style-loader', ...extractTextPlugin.use]
+        use: buildOptions.extractCss ? ExtractTextPlugin.extract(extractTextPlugin) : [{
+          loader: 'style-loader',
+          options: {
+            convertToAbsoluteUrls: cssSourceMap,
```

Comment:
This should not be needed.  From looking at [this code](https://github.com/webpack-contrib/style-loader/blob/c9707f12ff46f8e142607e0c43a0939cc76e48f8/lib/addStyles.js#L348), the option is automatically enabled if the stylesheet has a source map and the setting is undefined.